grunt_tasks/server.js

```diff
@@ -44,7 +44,8 @@ module.exports = function (grunt) {
     grunt.config.merge({
         shell: {
             readServerConfig: {
-                command: 'env python config_parse.py girder/conf/girder.local.cfg',
+                command: 'env python config_parse.py ' +
+                         grunt.config.get('girderDir') + '/conf/girder.local.cfg',
```

Comment:
This task is somewhat problematic for a number of reasons.  Is there a way we can make the static and api paths a runtime parameter?  The only place I found that these are used is in the rendered `testEnv.html` file.  If that is the case, I think we should be able to remove this from the build.